backend/src/spotify.py

```diff
@@ -0,0 +1,166 @@
+import os
+from typing import List
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+from src.album import Album
+
+from src.text_formatting import format_ms_as_mins_and_secs
+
+scope = [
+    "user-library-read",
+    "user-read-currently-playing",
+    "user-read-playback-state",
+    "playlist-modify-public",
+    "playlist-modify-private",
+    "playlist-read-private",
+    "playlist-read-collaborative",
+    "playlist-modify-private",
+    "playlist-modify-public",
+]
+
+
+class PlaybackInfo:
```

Comment:
I'm assuming this is for functionality yet to come

Reply:
Ah yeah, that's a spoiler from some playing around I was doing before starting this project. I'd remove it for YAGNI principles but I'm planning to add stuff onto it right away.